github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -363,21 +363,20 @@ presubmits:
                   - "true"
               topologyKey: kubernetes.io/hostname
       containers:
-      - image: kubevirtci/bootstrap:v20201119-a5880e0
+      - image: quay.io/kubevirtci/golang:v20210316-d295087
+        env:
+          - name: "TARGET"
+            value: "kind-k8s-sriov-1.17.0"
+          - name: "GIMME_GO_VERSION"
+            value: "1.13.8"
         command:
         - "/usr/local/bin/runner.sh"
         - "/bin/bash"
         - "-c"
-        - >
-            apt update &&
-            apt install gettext-base -y &&
-            wget https://dl.google.com/go/go1.13.8.linux-amd64.tar.gz -nv -S &&
-            tar -xzf go1.13.8.linux-amd64.tar.gz -C /usr/local/ &&
-            export PATH=/usr/local/go/bin:$PATH &&
-            export KUBEVIRT_PROVIDER=kind-k8s-sriov-1.17.0 &&
-            export TARGET=kind-k8s-sriov-1.17.0;
-            trap "echo teardown && make cluster-down" EXIT ERR SIGINT SIGTERM;
-            automation/test.sh;
+        - |
+            set -e
+            trap "echo teardown && make cluster-down" EXIT ERR SIGINT SIGTERM
```

Comment:
since i removed `export KUBEVIRT_PROVIDER=kind-k8s-sriov-1.17.0`
the trap here tries to run the wrong cluster-down
we have already a cluster-down in automation/test.sh
so i will remove this one ok ?
alternatively i can readd the `export KUBEVIRT_PROVIDER=kind-k8s-sriov-1.17.0`